WireCalling/Tests/WireCallingUITests/MeetingsListViewModelTests.swift

```diff
@@ -0,0 +1,147 @@
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import Testing
+import WireCallingDomain
+import WireCallingUI
+
+@Suite("MeetingsListViewModel Tests")
+struct MeetingsListViewModelTests {
+    let calendar = Calendar.current
+    let dateFormatter: DateFormatter
+    var viewModel: MeetingsListViewModel!
+    let currentDate = Date()
+
+    init() {
+        self.dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd HH:mm"
+        dateFormatter.timeZone = TimeZone(abbreviation: "CEST")!
+
+        let meetings: [Meeting] = [
+            Meeting(
+                id: UUID(),
+                title: "Ongoing Meeting",
+                start: calendar.date(byAdding: .minute, value: -5, to: currentDate)!,
+                end: calendar.date(byAdding: .minute, value: 60, to: currentDate)!
+            ),
+            Meeting(
+                id: UUID(),
+                title: "Today Meeting 1",
+                start: calendar.date(byAdding: .hour, value: 1, to: currentDate)!,
+                end: calendar.date(byAdding: .minute, value: 75, to: currentDate)!
+            ),
+            Meeting(
+                id: UUID(),
+                title: "Today Meeting 2",
+                start: calendar.date(byAdding: .hour, value: 2, to: currentDate)!,
+                end: calendar.date(byAdding: .minute, value: 135, to: currentDate)!
+            ),
+            Meeting(
+                id: UUID(),
+                title: "Tomorrow Meeting",
+                start: calendar.date(byAdding: .day, value: 1, to: calendar.startOfDay(for: currentDate))!,
+                end: calendar.date(byAdding: .minute, value: 1445, to: currentDate)!
+            ),
+            Meeting(
+                id: UUID(),
+                title: "Past Today Meeting",
+                start: calendar.date(byAdding: .hour, value: -1, to: currentDate)!,
+                end: calendar.date(byAdding: .minute, value: -30, to: currentDate)!
+            ),
+            Meeting(
+                id: UUID(),
+                title: "Yesterday Meeting",
+                start: calendar.date(byAdding: .day, value: -1, to: calendar.startOfDay(for: currentDate))!,
+                end: calendar.date(byAdding: .hour, value: -6, to: currentDate)!
+            )
+        ]
+
+        self.viewModel = MeetingsListViewModel(meetings: meetings, currentDate: currentDate)
+    }
+
+    @Test("Verify ongoing meetings are correctly filtered")
+    func testOngoingMeetings() {
```

Comment:
nitpick: Test methods using swift testing don't need the `test` prefix. 
```suggestion
    func ongoingMeetings() {
```

Reply:
no test anymore